lib/create-contentful-api.js

```diff
@@ -63,9 +64,61 @@ export default function createClientApi ({ http }) {
   const {wrapSpace, wrapSpaceCollection} = entities.space
   const {wrapUser} = entities.user
   const {wrapPersonalAccessToken, wrapPersonalAccessTokenCollection} = entities.personalAccessToken
-  const {wrapOrganizationCollection} = entities.organization
+  const {wrapOrganization, wrapOrganizationCollection} = entities.organization
   const {wrapUsageCollection} = entities.usage
 
+  /**
+   * Gets a collection of Organizations
+   * @memberof ContentfulClientAPI
+   * @return {Promise<OrganizationCollection>} Promise for a collection of Organizations
+   * @example
+   * const contentful = require('contentful-management')
+   *
+   * const client = contentful.createClient({
+   *   accessToken: '<content_management_api_key>'
+   * })
+   *
+   * client.getOrganizations()
+   * .then(result => console.log(result.items))
+   * .catch(console.error)
+   */
+  function getOrganizations () {
+    const baseURL = http.defaults.baseURL.replace('/spaces/', '/organizations/')
```

Comment:
you can't... because the default base URL has `spaces` hardcoded in it, so you'd end up with `contentful.com/spaces/organizations`

This pattern just follows how we deal with creating personal access tokens, there they replace spaces for `/users` as well (see `lib/create-contentful-api.js` -> `createPersonalAccessToken`)

Reply:
true, we'd have to go into sdk-core and update the http client to allow for creating an org-scoped http client instead of space scoped.

I'm fine with this workaround (even prefer it)